
Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-credential-initializer-bx45z[0m
{"level":"info","ts":1593175159.435695,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1593175159.436442,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-working-dir-initializer-h9mqb[0m
{"level":"info","ts":1593175160.968785,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1593175160.97382,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-place-tools[0m

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-source-ablazleon-jxpy-pr-2-pr-build-fdnjx-lb78x[0m
{"level":"info","ts":1593175170.4694967,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1593175173.6572335,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/*********/**py.git @ master in path /workspace/source"}

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:6fa97baa7d3eadc9892ee61f4084edb285dd97f4,2:eab011ea445d3607d6e0375677403c7da9d16bde
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git fetch origin --unshallow eab011ea445d3607d6e0375677403c7da9d16bde: 6fa97baa7d3eadc9892ee61f4084edb285dd97f4:
DEBUG: ran git fetch --unshallow origin eab011ea445d3607d6e0375677403c7da9d16bde: 6fa97baa7d3eadc9892ee61f4084edb285dd97f4: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 6fa97baa7d3eadc9892ee61f4084edb285dd97f4
DEBUG: ran git reset --hard 6fa97baa7d3eadc9892ee61f4084edb285dd97f4 in 
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: JX_LOG_LEVEL=error LC_ALL=C git merge eab011ea445d3607d6e0375677403c7da9d16bde
DEBUG: ran git merge eab011ea445d3607d6e0375677403c7da9d16bde in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git log --format=%H%x1f%an%x1f%ae%x1f%cn%x1f%ce%x1f%s%n%b%x1e 6fa97baa7d3eadc9892ee61f4084edb285dd97f4..HEAD
Merged SHA eab011ea445d3607d6e0375677403c7da9d16bde with commit message 'Update app.py' into base branch master

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-python-unittest[0m

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-build-container-build[0m
E0626 12:39:38.704185       9 aws_credentials.go:77] while getting AWS credentials NoCredentialProviders: no valid providers in chain. Deprecated.
	For verbose messaging see aws.Config.CredentialsChainVerboseErrors
[36mINFO[0m[0001] Using dockerignore file: /workspace/source/.dockerignore 
[36mINFO[0m[0001] Retrieving image manifest python:onbuild     
E0626 12:39:39.150269       9 metadata.go:154] while reading 'google-dockercfg' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg
E0626 12:39:39.154098       9 metadata.go:166] while reading 'google-dockercfg-url' metadata: http status code: 404 while fetching url http://metadata.google.internal./computeMetadata/v1/instance/attributes/google-dockercfg-url
[36mINFO[0m[0002] Retrieving image manifest python:onbuild     
[36mINFO[0m[0003] Built cross stage deps: map[]                
[36mINFO[0m[0003] Retrieving image manifest python:onbuild     
[36mINFO[0m[0004] Retrieving image manifest python:onbuild     
[36mINFO[0m[0005] Executing 3 build triggers                   
[36mINFO[0m[0005] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:dc2a839bd6d525fd9783a6852939774c55dc14024a49397cd18ef840aae363e1... 
[36mINFO[0m[0006] Using caching version of cmd: COPY requirements.txt /usr/src/app/ 
[36mINFO[0m[0006] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:8bf2abf0a84228eb1cc03999f3ee3ce7361230c16e28eae7af74285f6291b7d4... 
[36mINFO[0m[0006] Using caching version of cmd: RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0006] Checking for cached layer gcr.io/envjenkinsk8udacitycapstone/cache:65ba69d94d4818c7e0807d0dd26a52c5499de2ba95d3acb5d0b38db2f3e7ad92... 
[36mINFO[0m[0007] No cached layer found for cmd COPY . /usr/src/app 
[36mINFO[0m[0007] cmd: EXPOSE                                  
[36mINFO[0m[0007] Adding exposed port: 8080/tcp                
[36mINFO[0m[0007] Unpacking rootfs as cmd COPY . /usr/src/app requires it. 
[36mINFO[0m[0027] COPY requirements.txt /usr/src/app/          
[36mINFO[0m[0027] Found cached layer, extracting to filesystem 
[36mINFO[0m[0027] RUN pip install --no-cache-dir -r requirements.txt 
[36mINFO[0m[0027] Found cached layer, extracting to filesystem 
[36mINFO[0m[0028] COPY . /usr/src/app                          
[36mINFO[0m[0028] Resolving 137 paths                          
[36mINFO[0m[0028] Taking snapshot of files...                  
[36mINFO[0m[0028] Pushing layer gcr.io/envjenkinsk8udacitycapstone/cache:65ba69d94d4818c7e0807d0dd26a52c5499de2ba95d3acb5d0b38db2f3e7ad92 to cache now 
[36mINFO[0m[0028] COPY requirements.txt .                      
[36mINFO[0m[0028] Resolving 1 paths                            
[36mINFO[0m[0028] Taking snapshot of files...                  
[36mINFO[0m[0028] Pushing layer gcr.io/envjenkinsk8udacitycapstone/cache:d5138a587f6757be55121e3ea0295fed359b604abff7e255eeb07e1ac6883a12 to cache now 
[36mINFO[0m[0028] ENV PORT 8080                                
[36mINFO[0m[0028] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0028] EXPOSE 8080                                  
[36mINFO[0m[0028] cmd: EXPOSE                                  
[36mINFO[0m[0028] Adding exposed port: 8080/tcp                
[36mINFO[0m[0028] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0028] ENTRYPOINT ["python"]                        
[36mINFO[0m[0028] No files changed in this command, skipping snapshotting. 
[36mINFO[0m[0028] CMD ["app.py"]                               
[36mINFO[0m[0028] No files changed in this command, skipping snapshotting. 

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-postbuild-post-build[0m
no CVE provider running in the current ** namespace so skip adding image to be analysed

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-make-preview[0m
sed -i -e "s/version:.*/version: 0.0.0-SNAPSHOT-PR-2-1/" Chart.yaml
sed -i -e "s/version:.*/version: 0.0.0-SNAPSHOT-PR-2-1/" ../*/Chart.yaml
sed -i -e "s|repository:.*|repository: gcr.io\/envjenkinsk8udacitycapstone\/**py|" values.yaml
sed -i -e "s/tag:.*/tag: 0.0.0-SNAPSHOT-PR-2-1/" values.yaml
echo "  version: 0.0.0-SNAPSHOT-PR-2-1" >> requirements.yaml
** step helm build
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: storage.googleapis.com https://storage.googleapis.com/chartmuseum.jenkins-x.io
Successfully added Helm repository storage.googleapis.com.
Adding missing Helm repo: jenkins-x-chartmuseum http://jenkins-x-chartmuseum:8080
Successfully added Helm repository jenkins-x-chartmuseum.
WARNING: No $CHART_REPOSITORY defined so using the default value of: http://jenkins-x-chartmuseum:8080
Adding missing Helm repo: chartmuseum.jenkins-x.io http://chartmuseum.jenkins-x.io
Successfully added Helm repository chartmuseum.jenkins-x.io.

Showing logs for build [32mablazleon-jxpy-pr-2-pr-build-fdnjx-1[0m stage [32mfrom-build-pack[0m and container [32mstep-promote-jx-preview[0m
Creating a preview
Found commit author match for: ********* with email address: adrian.blazquez.leon@alumnos.upm.es
Created environment *********-**py-pr-2
Namespace **-*********-**py-pr-2 created 
expose:
  Annotations:
    helm.sh/hook: post-install,post-upgrade
    helm.sh/hook-delete-policy: hook-succeeded
  config:
    domain: 34.91.102.68.nip.io
    exposer: Ingress
    http: "true"
preview:
  image:
    repository: gcr.io/envjenkinsk8udacitycapstone/**py
    tag: 0.0.0-SNAPSHOT-PR-2-1

Updating PipelineActivities *********-**py-pr-2-1 which has status Running
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 783.034151ms
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 1.077829579s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 2.409014871s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 2.690153652s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 6.912008504s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 502 502 Bad Gateway, delaying for: 4.011801539s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 16.803924065s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 9.553100503s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 17.255077299s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 29.928448281s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 43.452942397s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 51.708328829s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 1m23.66170376s
WARNING: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable, delaying for: 34.457459937s
error: error checking if preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io is available: preview application http://**py.**-*********-**py-pr-2.34.91.102.68.nip.io not available, error was 503 503 Service Temporarily Unavailable
[31m
Pipeline failed on stage 'from-build-pack' : container 'step-promote-jx-preview'. The execution of the pipeline has stopped.[0m
